### ISSUE TITLE
Controller Volume Slider Separated

### ISSUE THREAD

**Issue by DisarmedSpy** [Paliverse/DualSenseX#45]
One small request is perhaps adding a volume slider for the haptics that isn't tied to the computers volume. While the slider already there is fine, there needs to be a little bit more customizability and making it separate from the regular volume. ie. With Voicemeeter I have the controller set to -7 volume, as at 0 or above it just feels TOO strong, as well as the ability to change my computers volume without affecting that -7. This is my only request for this new feature.

In short: Could you make it so the volume of the computer doesn't affect how powerful the advanced haptics are on the controller, as well as the ability to change how powerful the advanced haptics on the controller are in the first place as they still feel VERY powerful even when the volume is as low as I can make it with the sliders.

**Comment by DisarmedSpy** [comment]
Also no idea how to add things such as "Request" to my title so I apologize

**Comment by Paliverse** [comment]
Hello, currently adding this for the next update.

**Comment by DisarmedSpy** [comment]
> Hello, currently adding this for the next update.

Excellent! Looking forward to it (And finally being able to officially get rid of Voicemeeter)
